packages/@okta/vuepress-site/docs/reference/api/asa/clients/index.md

```diff
@@ -0,0 +1,263 @@
+
+
+### List ASA Clients
+
+<ApiOperation method="GET" url="https://app.scaleft.com/v1/teams/${team_name}/clients" />
+This endpoint returns a list of ASA Clients. By default returns the ASA Clients for the requesting ASA User. Query options can be provided to filter by ASA User or state.
+
+This endpoint requires one of the following roles: `access_admin`, `reporting_user`, `access_user`.
+
+#### Request path parameters
+
+| Parameter | Type        | Description   |
+| --------- | ----------- | ------------- |
+| `team_name`   | string | The name of your Team. |
+
+
+#### Request query parameters
+
+| Parameter | Type   | Description |
+| --------- | ------------- | -------- |
+| `all`   |  boolean | (Optional) When `true`, returns all Clients for the Team. |
+| `count`   |  number | (Optional) The number of objects per page. |
+| `descending`   |  boolean | (Optional) The object order. |
+| `offset`   |  string | (Optional) The page offset. |
+| `prev`   |  boolean | (Optional) The direction of paging. |
+| `state`   |  string | (Optional) One of `ACTIVE`, `PENDING`, `DELETED`. |
+| `username`   |  string | (Optional) List Clients assigned to a single User. An empty string returns unassigned Clients. |
+
+
+#### Request body
+
+This endpoint has no request body.
+
+#### Response body
+This endpoint returns a list of objects with the following fields and a `200` code on a successful call.
+| Properties | Type        | Description          |
+|----------|-------------|----------------------|
+| `description`   | string | A description of the ASA Client. |
+| `encrypted`   | boolean | A boolean reflecting whether `sft` was able to determine that the ASA Client was encrypted. |
+| `hostname`   | string | The hostname of the ASA Client. |
+| `id`   | string | The UUID of the ASA Client. |
+| `os`   | string | The OS of the ASA Client device. |
+| `state`   | string | The state of the ASA Client. One of: `ACTIVE`, `PENDING`, or `DELETED`. |
+| `user_name`   | string | The ASA User to whom this ASA Client belongs. |
+
+#### Usage example
+
+##### Request
+
+```bash
+curl -v -X GET \
+-H "Authorization: Bearer ${jwt}" \
+https://app.scaleft.com/v1/teams/${team_name}/clients
+```
+
+##### Response
+
+```json
+{
+	"list": [
+		{
+			"deleted_at": null,
+			"description": "Work laptop",
+			"encrypted": true,
+			"hostname": "LightInAugust",
+			"id": "9c199afe-0ca5-427a-baac-c4341707d82b",
+			"os": "macOS 10.14.6",
+			"state": "PENDING",
+			"user_name": "Jason.Compson.IV"
+		},
+		{
+			"deleted_at": null,
+			"description": "Personal laptop",
+			"encrypted": true,
+			"hostname": "Absalom",
+			"id": "5e9d8179-2fca-4905-8405-3cd3e0c5280e",
+			"os": "macOS 10.14.6",
+			"state": "ACTIVE",
+			"user_name": "Jason.Compson.IV"
+		}
+	]
+}
+```
+### Fetch an ASA Client
+
+<ApiOperation method="GET" url="https://app.scaleft.com/v1/teams/${team_name}/clients/${client_id}" />
+This endpoint fetches a single ASA Client associated with the ID in the path.
```

Comment:
```suggestion
This endpoint fetches a single ASA Client with the specified Client ID on the specified Team.
```

Not sure if "on the Team" is the right phrase, but we should mention the other variable in the path.